tests/test_output.py

```diff
@@ -1204,8 +1202,6 @@ def test_result_show_backup(self, capsys):
         assert ext_info["status"] in out
         assert str(ext_info["end_time"]) in out
         assert ext_info["systemid"] in out
-        for name, _, location in ext_info["tablespaces"]:
-            assert "{:<21}: {}".format(name, location) in out
```

Comment:
This seems an unrelated change -- it's taking out the unit tests for `barman show-backup` command.

I'd say to revert this removal of lines here.